.github/workflows/pr-realm-js.yml

```diff
@@ -612,9 +612,7 @@ jobs:
       - build-ts
       - generate-jsi
       - prebuild-android
-    env:
-      SPAWN_LOGCAT: true
-    runs-on: macos-latest-large
+    runs-on: ubuntu-latest
```

Comment:
I'm also switching to ubuntu, as that allows hardware acceleration and is supposedly faster (see https://github.com/realm/realm-js/pull/6444 and https://github.com/realm/realm-dart/commit/2d93a4f3 for context).

Reply:
Scrap that. This results in a `clang++` crash (likely caused by the runner [lacking disk space](https://groups.google.com/g/android-llvm/c/YkWDdy_Khu8))

>  PLEASE submit a bug report to https://github.com/android-ndk/ndk/issues and include the crash backtrace.
  clang++: error: unable to execute command: Bus error (core dumped)
  clang++: error: linker command failed due to signal (use -v to see invocation)
  ninja: build stopped: subcommand failed.